tests/test_utils.py

```diff
@@ -41,6 +41,14 @@ def test_pd_io_xlsx_multi():
         _obs = obs[k]
         pdt.assert_frame_equal(_obs, _exp)
 
+def test_ispanda():
```

Comment:
This test should:
- Probably be named `test_pd_write` since `pd_write` is the name of the function being tested.
- Use the [pytest `tmp_path` fixture](https://docs.pytest.org/en/latest/tmpdir.html#the-tmp-path-fixture). As it is, you will notice that 'test.csv' is created in whichever directory you run `pytest`, which is not desirable; it confuses the user via git.